tools/client/README.md

```diff
@@ -14,15 +14,13 @@ For help:
 Request resource from KBS:
 
 ```shell
-./client --resource-uri kbs://127.0.0.1:8080/<repository>/<type>/<tag>
+./client --kbs-uri https://127.0.0.1:8080/ --resource-path /<repository>/<type>/<tag>
```

Comment:
Do we need the `/` after `https://127.0.0.1:8080` ?

Reply:
Fixed, good catch.